dev.sh

```diff
@@ -1,11 +1,21 @@
 #!/bin/bash
 
-nodemon openshift-auth-proxy.js \
-  --tls-cert test-cert.pem \
-  --tls-key test-key.pem \
-  --openshift-master https://localhost:8443 \
-  --openshift-public-master https://localhost:8443 \
-  --openshift-ca /var/lib/openshift/openshift.local.config/master/ca.crt \
-  --client-id integration-services \
-  --client-secret 12345 \
+# set or override any args:
+# sh dev.sh --master-url=...
+
+if [[ ! -d ./secret ]]; then
+	cp -r ./secret.example ./secret
+fi
+# need the master CA too
+if [[ ! -e ./secret/master-ca ]]; then
+	echo "copy your master CA file to ./secret/master-ca to use this script"
+	exit 1
+fi
+
+node openshift-auth-proxy.js \
```

Comment:
I don't want to have to add nodemon to the dependencies. But I suppose this script will never be used in a container...